### ISSUE TITLE
Instagram registration does not work with Mypal or FF52

### ISSUE THREAD

**Issue by wyxchari** [Feodor2/Mypal#37]
The registration in Instagram with Mypal 28.4.0 in XP always gives error. After, the mail and user are blocked as already used.
Instagram registration with Mypal in W10 does not work and does the same.
Instagram registration with Firefox 52.9.0 in XP does not work and does the same.
Instagram registration with Firefox 65.0.2 works fine in W10. After registration with FF65 you can already log in with Mypal / FF52.

**Comment by Feodor2** [comment]
Just have passed through registration at the moment. Yes it was blocked my first name, so i changed it e until Feodor123234214124 has been accepted. Do you want to say that FF65 with the first user name can do fine, because i do not have it to check.

**Comment by wyxchari** [comment]
With Mypal or FF52, when registering on Instagram, we put the data, ask the age and error always occurs and you can not register.
It does not matter if the mail and user is registered on Instagram and can no longer be used to re-register. The problem is that the registration never ends.
After registering with FF65, we can login with Mypal. What fails is the registration with Mypal, not the login.

**Comment by Feodor2** [comment]
Please spell literally that red text in english.

**Comment by wyxchari** [comment]
Sorry. I close the topic. It's not Mypal's fault.
The error in the Instagram registration is due to the algorithm they use to detect attacks. They use the IP, the mail and the device to post the number of times someone has registered. The limit is 5 times. I think I have reached the limit because I registered, forgot the password and returned to register with another mail once or twice a year. Instagram recovers the times you have registered from the same device. That's why I was able to do it from FF65 on another computer even using the same IP. I will take note of this so that I do not forget. Thanks for the help.
Mypal forever !!!!!!!!!!!!!!!!!!!!!!!!!!
